src/View/SSTemplateParser.peg

```diff
@@ -1116,27 +1107,6 @@ class SSTemplateParser extends Parser implements TemplateParser
         }
     }
 
-    /**
-     * This is an open block handler, for the <% debug %> utility tag
-     */
-    function OpenBlock_Handle_Debug(&$res)
-    {
-        if ($res['ArgumentCount'] == 0) {
-            return '$scope->debug();';
-        } elseif ($res['ArgumentCount'] == 1) {
-            $arg = $res['Arguments'][0];
-
-            if ($arg['ArgumentMode'] == 'string') {
-                return 'Debug::show('.$arg['php'].');';
-            }
-
-            $php = ($arg['ArgumentMode'] == 'default') ? $arg['lookup_php'] : $arg['php'];
-            return '$val .= Debug::show('.str_replace('FINALGET!', 'cachedCall', $php ?? '').');';
-        } else {
-            throw new SSTemplateParseException('Debug takes 0 or 1 argument only.', $this);
-        }
-    }
```

Comment:
This hasn't worked since CMS 4! Rather than fixing it, I've just opted to remove it since clearly nobody cares about it.